platform/commonUI/browse/bundle.js

```diff
@@ -241,7 +247,7 @@ define([
                 {
                     "key": "items",
                     "name": "Items",
-                    "cssClass": "icon-thumbs-strip",
+                    "cssclass": "icon-thumbs-strip",
```

Comment:
@evanohara This should be `cssClass`, not `cssclass`. Not sure how this happened, could have been rebase.